### PR TITLE
docs(*): change 'npm i' to 'npm install'

### PR DESCRIPTION
**Pull request by sukvvon**
## Summary

* Since this is an official document, I determined that using `npm install` instead of the abbreviation `npm i` is more appropriate, so I made the change.
* In the official homepage, both `npm install` and `npm i` are used, so I standardizied them.

## Check List

- [x] `pnpm run prettier` for formatting code and docs

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -5,7 +5,7 @@
 
 <br>
 
-visit [jotai.org](https://jotai.org) or `npm i jotai`
```
Comment (dai-shi) [inline readme.md:8]: At least, this was very intentional.

**Review by dai-shi — COMMENTED** [review]
I'm fine with this change, but I would like to hear from someone else. How about @dmaskasky and @sandren ?

File: readme.md (hunk 1 above)
Comment (sandren) [reply readme.md:8]: Yes, let's keep the shorthand here at least. 🙂

File: readme.md [hunk 2]
```diff
@@ -5,7 +5,7 @@
...
+visit [jotai.org](https://jotai.org) or `npm install jotai`
```
Comment (dai-shi) [inline readme.md:8]:
```suggestion
visit [jotai.org](https://jotai.org) or `npm i jotai`
```

**Comment by dai-shi** [comment]
@sukvvon When you have a chance, I'd appreciate if you can look into the flaky test issue. We've been struggling with it with no clues yet.

**Comment by sukvvon** [comment]
> @sukvvon When you have a chance, I'd appreciate if you can look into the flaky test issue. We've been struggling with it with no clues yet. https://github.com/pmndrs/jotai/actions/runs/12385559467/job/34572020212?pr=2865

@dai-shi ok, i'll check it

**Review by dai-shi — APPROVED** [review]
LGTM

**Pull request merged**